module build failed

I've tried to run the example-app on two workstations (both OSX) and getting the following error.

ERROR in ./src/main.browser.ts
Module build failed: Error: Cannot resolve module '@angular2-material/list' in /Users/tahir/ngrx-examples/example-app/src
    Required in /Users/tahir/ngrx-examples/example-app/src/app.ts
    at ResolutionError.Error (native)
    at new ResolutionError (/Users/tahir/ngrx-examples/example-app/node_modules/awesome-typescript-loader/dist.babel/deps.js:515:88)
    at /Users/tahir/ngrx-examples/example-app/node_modules/awesome-typescript-loader/dist.babel/deps.js:340:37
    at run (/Users/tahir/ngrx-examples/example-app/node_modules/core-js/modules/es6.promise.js:89:22)
    at /Users/tahir/ngrx-examples/example-app/node_modules/core-js/modules/es6.promise.js:102:28
    at flush (/Users/tahir/ngrx-examples/example-app/node_modules/core-js/modules/_microtask.js:18:9)
    at doNTCallback0 (node.js:407:9)
    at process._tickCallback (node.js:336:13)
 @ multi main
Child html-webpack-plugin for "index.html":
    chunk    {0} index.html 522 bytes [rendered]
        [0] ./~/html-webpack-plugin/lib/loader.js!./src/index.html 522 bytes {0} [built]
           factory:149ms building:14ms = 163ms
webpack: bundle is now VALID.
[default] Checking started in a separate process...
[default] /Users/tahir/ngrx-examples/example-app/src/app.ts:6:36
    Cannot find module '@angular2-material/list'.

I've node v5.x and npm 3.x.

--- a/package.json
+++ b/package.json
@@ -34,7 +34,7 @@
     "@angular2-material/button": "2.0.0-alpha.5-2",
     "@angular2-material/card": "2.0.0-alpha.5-2",
     "@angular2-material/core": "2.0.0-alpha.5-2",
-    "@angular2-material/grid-list": "2.0.0-alpha.5-2",
+    "@angular2-material/list": "2.0.0-alpha.5-2",
     "@angular2-material/icon": "2.0.0-alpha.5-2",
     "@angular2-material/input": "2.0.0-alpha.5-2",
     "@angular2-material/sidenav": "2.0.0-alpha.5-2",

